pom.xml

```diff
@@ -170,9 +170,9 @@
     <version.org.eclipse.aether>1.1.0</version.org.eclipse.aether>
     <version.org.eclipse.emf>2.6.0.v20100614-1136</version.org.eclipse.emf>
     <version.org.eclipse.emf.ecore.xmi>2.5.0.v20100521-1846</version.org.eclipse.emf.ecore.xmi>
-    <version.org.eclipse.jdt>3.19.0</version.org.eclipse.jdt>
+    <version.org.eclipse.jdt>3.18.0</version.org.eclipse.jdt>
```

Comment:
As far as I remember GWT (`gwt-dev`) include its own version of `jdt` directly in the jar and this version is only used by `drools-compiler`.
To avoid classpath clashing and similar, we shaded the version used by Drools ( [link](https://github.com/kiegroup/drools/blob/c7fba5bd3ffdacadfea2eabfb00e47467c463d94/drools-compiler/pom.xml#L207-L235) ) and after that change we had no other `jdt` upgrade problem like in the past.

I just noticed that together with Errai GWT 2.9 upgrade, it has been added an explicit dependency to `jdt` ( [link](https://github.com/errai/errai/commit/0ecd8fd5ed1920f8357a4d9989f968d79b429a9c) ) for example in `errai-codegen`. Is it possible in Errai to continue to use `jdt` provided by GWT? Or if not possible what about shade it so that it will not leak here?

We need to preserve the possibility to upgrade `jdt` for drools core when needed and this downgrade might break other stuff on Drools (and Kogito too).

/cc @mariofusco

Reply:
Ok I just noticed we are already shading it in errai ( [link](https://github.com/errai/errai/blob/master/errai-codegen/pom.xml#L142) ) so my only question is why this downgrade? :)

Reply:
align dependencies creates less problems, and the downgrade had no affect on runtimes

Reply:
I'm fine with this, my only concern is to avoid if possible to have to keep them aligned: we already had in the past the need to upgrade `ecj` for `drools-compiler` and I would like to preserve this possibility. Do you expect this upgrade will prevent us/increase the complexity to have different versions for GWT/drools-compiler in the future? If this is and will be possible I'm fine with this downgrade :)

Reply:
I'd expect this would be the last upgrade related to GWT in this stream, so keep it simple for now... won't prevent us to do better in future.

But for now I'd prefer the simplification :)